roles/network_plugin/calico/templates/cni-calico.conflist.j2

```diff
@@ -28,6 +28,16 @@
         "subnet": "usePodCidr"
       },
 {% endif %}
+{% if (calico_feature_control is defined) and (calico_feature_control|length > 0) %}
+{% if calico_feature_control|length > 0 %}
```

Comment:
Remove this line. There is 2 if's

Reply:
Sorry for this guy, I made the change like a robot!